catalog/async/catalog-async-data/src/main/java/org/codice/ddf/catalog/async/data/impl/ProcessResourceImpl.java

```diff
@@ -183,8 +202,11 @@ public boolean isModified() {
   }
 
   @Override
-  public void close() {
-    IOUtils.closeQuietly(inputStream);
+  public synchronized void close() {
+    try {
```

Comment:
❓ Doesn't `IOUtils.closeQuietly()` work here around the closer? (same in the above class)